core_contracts/dex/dex.py

```diff
@@ -551,11 +596,30 @@ def is_locking_pool(self, id: int) -> bool:
         Internal function, mapped to mainnet pool IDs. Will be deprecated upon the release
         of continuous rewards.
         """
-        if id < FIRST_NON_BALANCED_POOL or id == USDS_BNUSD_ID or id == IUSDT_BNUSD_ID:
+
+        continuous_rewards_launched = (self._continuous_rewards_day.get() > self._current_day.get())
+        restricted_pool_id = (id < FIRST_NON_BALANCED_POOL or id == USDS_BNUSD_ID or id == IUSDT_BNUSD_ID)
+
+        if  not restricted_pool_id or continuous_rewards_launched:
```

Comment:
The 'not' here makes this return the opposite of what it currently does. Maybe it should return True on this condition?

Reply:
I actually fixed this on sejong a week ago and forgot to push, adding now.